draft/spec/index.html

```diff
@@ -180,24 +180,30 @@ <h2>Introduction</h2>
+          The OCFL initiative arose from a need to have well-defined application-independent file management within
+          digital repositories.
         </p>
         <p>
-           A general observation is that the contents of a digital repository &#8212; that is, the digital files and
-           metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
-           accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
-           applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
-           Thus, transitions between application-specific methods of file management to support software upgrades and
-           replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
-           in support of the short-term, ephemeral software.
+          A general observation is that the contents of a digital repository &#8212; that is, the digital files and
+          metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
+          accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
+          applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
+          Thus, transitions between application-specific methods of file management to support software upgrades and
+          replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
+          in support of the short-term, ephemeral software.
         </p>
         <p>
-           By providing a specification for the file and directory layout on disk or in an object store, OCFL is an
-           attempt at reducing, or even eliminating, the need for these transitions. As an application-independent
-           specification, conforming applications will natively "understand" the underlying file structure without
-           needing to first transition these contents to their own format.
+          By providing a specification for the file and directory layout on disk or in an object store, the OCFL is an
+          attempt at reducing, or even eliminating, the need for these transitions. As an application-independent
+          specification, conforming applications will natively "understand" the underlying file structure without
+          needing to first transition these contents to their own format.
          </p>
+        <p>
+          While digital repository content changes relatively slowly, it is necessary to be able to track changes to 
+          digital objects. Within the file and directory layout specification, OCFL provides a simple structure to 
+          efficiently capture versions of object contents so that all previous states of an object may be recovered
+          and examined.
```

Comment:
Should we also note that the specification makes no recommendations on what constitutes a version, that is left to the institution to decide.

Reply:
I don't think we need to put that in the need section -- we already have it later https://ocfl.io/draft/spec/#object-spec

> A central feature of the OCFL specification is support for versioning. This recognizes that digital objects will change over time, through new requirements, fixes, updates, or format shifts. The specification takes no position on what constitutes a version or a versionable action, but it is recommended that implementers have a clear position on this within their local storage policies.

Reply:
I'll approve and contemplate more language here then.  While we talk a great deal in this section about the need for application independent storage, this paragraph in contrast seems weak given so many folks see versioning as the strength over something like BagIT (which could also be used as application independent storage and often is).